### PR TITLE
WHOSE FOOTPRINTS ARE THESE

### PR DESCRIPTION
**Pull request by Iamgoofball**

### CODE REVIEW

**Comment by tortellinitony** [comment]
*Whose

Learn English, please.

**Comment by Iamgoofball** [comment]
@tortellinitony go sneak around military bases in the snow more

**Comment by GunHog** [comment]
Sometimes your meme PRs deserve to be merged. This is one of those rare times! I am encouraged to download an emulator and play it again! (Yes, I have a Playstation and a copy of the game, so it is legal!)

**Comment by Iamgoofball** [comment]
i assume you are discussing emulating a
# legal linux distro
correct?

**Comment by GunHog** [comment]
I choose to withhold further comment until my lawyer @Robustin can advise me properly.

**Comment by RemieRichards** [comment]
if your emulator's bios is sourced from your own playstation, and you own the game, it's legal.
otherwise it is not legal, but you are also an individual, and presumably not planning on distributing it which makes you a very expensive and poor value target for a lawsuit.

tl;dr you're probably fine.

**Comment by Iamgoofball** [comment]
or if you use an alternate BIOS like the phoom case

**Comment by RemieRichards** [comment]
well in 90% of emulation cases there's no such thing as an "alt bios", the playstation is an exception though I guess.

**Review by Cyberboss — APPROVED** [review]

**Comment by Anonmare** [comment]
https://youtu.be/JjPpX1g1juo?t=22s

**Comment by Militaires** [comment]
https://www.youtube.com/watch?v=JjPpX1g1juo
